### ISSUE TITLE
While fit.model ValueError:

### ISSUE THREAD

**Issue by elyorman** [ankurbhatia24/MULTIMODAL-EMOTION-RECOGNITION#4]
![image](https://user-images.githubusercontent.com/60263344/91111474-8f8ee180-e6bb-11ea-943f-64efdbce8bea.png)

Facing this error to train the model 
I don't understand why only one input is being given here not three?

**Comment by ankurbhatia24** [comment]
> Facing this error to train the model
> I don't understand why only one input is being given here not three?

Yes, you are right. If you want you can submit a PR or I'll update it in some time.

**Comment by LinksYuan** [comment]
I encounter the same problem, how should I solve it.
